CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Refactor: use normal Authorization header instead of X-Authorization to connect to the API
 * Add local seed data option to init-database
 * Move migrations script to npm run migrate-database
+* Removed the moment lib dependency; moment is deprecated
```

Comment:
Not neccessarly, moment is done, its is still maintained. They have a list of recommended alternatives: https://momentjs.com/docs/#/-project-status/recommendations/